docs/reference/edot-collector/troubleshooting/collector-oomkilled.md

```diff
@@ -0,0 +1,109 @@
+---
+navigation_title: EDOT Collector is out of memory
+description: Diagnose and resolve out-of-memory issues in the EDOT Collector using Go’s Performance Profiler.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# EDOT Collector is out of memory
+
+If your EDOT Collector is being terminated with an `OOMKilled` status, this usually indicates a memory leak or sustained memory pressure. You can use the Performance Profiler (`pprof`) tool to collect and analyze memory profiles, helping you identify the root cause.
```

Comment:
```suggestion
If your EDOT Collector terminates with an `OOMKilled` status, this usually indicates a memory leak or sustained memory pressure. You can use the Performance Profiler (`pprof`) extension to collect and analyze memory profiles, helping you identify the root cause of the issue.
```

The Collector has the concept of extensions for this kind of meta tasks -- better use the specific term as you do later in the doc.